ssl-impl/src/test/java/io/crate/protocols/postgres/ssl/SslConfigurationTest.java

```diff
@@ -87,7 +87,6 @@ public void testTrustStoreLoadingFail() throws Exception {
         settingsBuilder.put(SslConfigSettings.SSL_TRUSTSTORE_PASSWORD_SETTING_NAME, "wrongpassword");
 
         TrustStoreSettings.tryLoad(settingsBuilder.build());
-        fail();
```

Comment:
My point is not that fail is needed. It's about changing code purely because of style. If you change because of a bugs or unwanted behavior I don't mind. Keep in mind that potentially you can also introduce bugs with such tiny changes.